Makefile

```diff
@@ -206,6 +206,18 @@ ifeq (export-fs,$(firstword $(MAKECMDGOALS)))
   CID:=$(shell docker create $(IMAGE_EXPORT))
 endif
 
+.PHONY: print-var-image-name
+print-var-image-name:
+	@echo $(IMAGE_NAME)
+
+.PHONY: print-var-app-name
+print-var-app-name:
+	@echo $(RELEASE_NAME_APP)
+
+.PHONY: print-var-arg
+print-var-alg:
+	@echo $(ALG)
+
```

Comment:
I suppose these parts are unnecessary (useful during debug only).

Reply:
removed this part